test/scrabble_test.rb

```diff
@@ -5,8 +5,47 @@
 
 class ScrabbleTest < Minitest::Test
   def test_it_can_score_a_single_letter
-    skip
+    Scrabble.new.score("a")
```

Comment:
You do not need to call `Scrabble.new.score("a")` or `Scrabble.new.score("f")` here on lines 8 and 9, you are _effectively_ recalling these methods on lines 10 and 11 :)